README.md

```diff
@@ -24,6 +24,7 @@ Just invoke *logme* in your Fastfile action. Logme will return the commit messag
 | `to_revision` | End revision commit | false |
 | `from_revision` | From revision commit | false |
 | `message_regexp_filters` | Regexp filters like. ie '^MDM-\|^CTS:' | true |
+| `message_regexp_exclude_filters` | Regexp filters like. ie '^MDM-TST\|^CTS:UI' | true |
```

Comment:
This regex is simply a filter right? So it filters commits that should appear on the change log.

Why do we need two filters to filter the same thing?

Regular expressions allow negation of a pattern, so in the same regex you can have *including* and *excluding* patterns. In your example, the regex could be something like: `^MDM-|^NOJIRA-(?!.*UI)|` (haven't tested it though)